src/types/account_id.rs

```diff
@@ -53,3 +53,35 @@ impl AccountId {
         Self::from_str(owner_account_id).unwrap()
     }
 }
+
+pub fn is_implicit(account_id: &str) -> bool {
+    account_id.len() == 64
+        && account_id
+            .as_bytes()
+            .iter()
+            .all(|b| matches!(b, b'a'..=b'f' | b'0'..=b'9'))
+}
```

Comment:
No need to re-implement it, just re-export and use the [`get_account_type`](https://docs.rs/near-account-id/latest/near_account_id/struct.AccountId.html#method.get_account_type) method from the near-account-id.